plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -154,6 +164,21 @@ public boolean isRankActivated() {
     }
 
     /**
+     * Added to properly uncoercing.
```

Comment:
Generally public getters are required in order for `f:checkbox` to properly round-trip configuration. It is possible this was actually broken before your patch: if you forget the public getter, the checkbox will just be quietly unchecked every time you open the form. `JenkinsRule.configRoundTrip` (or, for Workflow steps, `StepConfigTester`) is the way to verify that your forms are actually loading the same data they save.